src/universal/modules/landing/ducks/auth.js

```diff
@@ -0,0 +1,27 @@
+const SET_AUTH_TOKEN = 'notifications/SET_AUTH_TOKEN';
```

Comment:
i could go either way. i stuck it in landing since that's where login happens, but `SET_TOKEN` is gonna be called from other modules, so moving it to the `redux` folder sounds good, too. 

PS, I gave it some thought, i say a socket handshake should refresh a token if it's older than a week. That way as long as they trigger socket activity, they'll have a fresh token. Also, since the refresh is handled on the server, we don't have to trust the clock on the client.